docs/update.md

```diff
@@ -1,58 +1,82 @@
-the following ones:
+You can upgrade Percona Operator for MySQL based on Percona XtraDB Cluster starting from the version 1.1.0.
 
-* the Operator;
-* [Custom Resource Definition (CRD)](operator.md),
-* Database Management System (Percona XtraDB Cluster).
+The upgrade process consists of these steps:
+
+* Upgrade the [Custom Resource Definition (CRD)](operator.md) and the Operator  
+* Upgrade the database (Percona XtraDB Cluster).
 
 !!! note
 
-    Additional steps are needed to upgrade database and Operator on [Red Hat Marketplace :octicons-link-external-16:](https://marketplace.redhat.com) or to upgrade Red Hat certified Operators on [OpenShift :octicons-link-external-16:](https://www.redhat.com/en/technologies/cloud-computing/openshift). See [this HOWTO](update_openshift.md) for details.
+    If you run the Operator on [Red Hat Marketplace :octicons-link-external-16:](https://marketplace.redhat.com) or you run Red Hat certified Operators on [OpenShift :octicons-link-external-16:](https://www.redhat.com/en/technologies/cloud-computing/openshift), you need to do additional steps during the upgrade. See [this HOWTO](update_openshift.md) for details.
 
 The list of recommended upgrade scenarios includes two variants:
 
 * Upgrade to the new versions of the Operator *and* Percona XtraDB Cluster,
 * Minor Percona XtraDB Cluster version upgrade *without* the Operator upgrade.
 
-## Upgrading the Operator and CRD
+## Upgrade the Operator and CRD
 
-!!! note
+### Considerations
 
-    The Operator supports **last 3 versions of the CRD**, so it is technically
-    possible to skip upgrading the CRD and just upgrade the Operator. If the CRD
-    is older than the new Operator version *by no more than three releases*, you
-    will be able to continue using the old CRD and even carry on Percona XtraDB
-    Cluster minor version upgrades with it. But the recommended way is to update
-    the Operator *and* CRD.
+1. The Operator version has three digits separated by a dot (`.`) in the format `major.minor.patch`. Here's how you can understand the version `1.17.0`:
 
-Only the incremental update to a nearest version of the
-Operator is supported (for example, update from 1.4.0 to 1.5.0). To update
-to a newer version, which differs from the current version by more
-than one, make several incremental updates sequentially.
+    * `1` is the major version 
+    * `17` is the minor version
+    * `0` is the patch version.     
 
-!!! warning
+    You can only upgrade the Operator to the nearest `major.minor` version (for example, from `1.16.x` to `1.17.x`).     
+
+    If the your current Operator version and the version you want to upgrade to differ by more than one minor version, you need to upgrade step by step. For example, if your current version is `1.15.x` and you want to move to `1.17.x`, first upgrade to `1.16.x`, then to `1.17.x`.    
+
+    Patch versions don't influence the upgrade, so you can safely move from `1.16.1` to `1.17.0`. 
```

Comment:
I think if we say `1.16.0` to `1.17.1`, it would be more understandable.

Reply:
We don't have `1.17.1` yet, that's why I used the versions available. I can amend to 1.15.1 to 1.16.1